### ISSUE TITLE
question about the loss function

### ISSUE THREAD

**Issue by WHUCSLWJ** [qqueing/DeepSpeaker-pytorch#8]
Could you tell me why you use the L2 distance rather than the Cosine Similarity proposed in the paper?

**Comment by WHUCSLWJ** [comment]
and why did you use torch.mean() instead of torch.sum() when compute the TripletMarginLoss. As I see in the paper, L=sigma(max(x,0)).
